src/main/java/org/candlepin/guice/CandlepinContextListener.java

```diff
@@ -426,17 +450,46 @@ protected void checkDbChangelog() {
                         .append(changeset.getFilePath())
                         .append('\n');
                 });
-
-                log.error(builder.toString());
-                log.error("Please update the database");
-                log.error("Aborting Candlepin initialization...");
-
-                throw new RuntimeException("The database is missing Liquibase changeset(s)");
+                // all levels above None need this
+                log.warn(builder.toString());
+
+                if (liquibaseOnStart.equals(DBManageLevel.REPORT.getName())) {
```

Comment:
This logic should live in the enum itself (if not using the enum's native `.valueOf` call), and then can be moved to the top of the method to avoid a bunch of work when we have an invalid argument.

```
DBManagementLevel dbmLevel = DBManagementLevel.valueOf(config.getString(DB_MANAGE_ON_START, "none"))

switch (dbmLevel) {
  case MANAGE:
    // do update bits here
    break;

  case HALT:
  case CHECK:
    // do status check & potentially halt here
    break;

  default:
  case NONE:
    // Do nothing, basically
}
```

You could also do individual blocks to have shared logic and logging in the case where we do want to manage it, it might be nice to see in the logs the changesets that appear to be missing before applying any update automatically.